readme.md

```diff
@@ -1,10 +1,13 @@
 # Frontend practice with catalog page using grid
+
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog_grid/)
+
+- [DEMO LINK](https://VladislavaT.github.io/layout_catalog_grid/)
```

Comment:
Do not add extra files in PR such as `readme` file
To add the file to a commit use `git add src/.`

Reply:
ok, I`ll note that!